src/ServiceControl/Bootstrapper.cs

```diff
@@ -56,7 +50,7 @@ public Bootstrapper(Settings settings, BusConfiguration configuration)
         private void Initialize()
         {
             var loggingSettings = new LoggingSettings(settings.ServiceName);
-            ConfigureLogging(loggingSettings);
+            RecordStartup(loggingSettings);
```

Comment:
`LoggingSettings` is not the problem, the issue is the `Settings` class which could log as part of it's ctor